raiden/transfer/state_change.py

```diff
@@ -73,6 +73,40 @@ def from_dict(cls, data: typing.Dict[str, typing.Any]) -> 'Block':
         )
 
 
+class ActionUpdateTransportTimestamp(StateChange):
+    """ Holds the last "timestamp" at which we synced
+    with the transport. Can be used later to filter
+    the messages which have not been processed.
+    """
+    def __init__(self, timestamp: str):
+        self.timestamp = timestamp
+
+    def __repr__(self):
```

Comment:
plz add types here and in __eq__ and __ne__. Unless somehow mypy automatically figures them out.